### PR TITLE
Feature/user authentication

### PR DESCRIPTION
**Pull request by duckduckboom**

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class SessionsController < ApplicationController
+  skip_before_action :authorized, only: [:new, :create, :welcome]
```
Comment (lewiscj97) [inline app/controllers/sessions_controller.rb:2]: Remove :new and :welcome

File: app/views/sessions/create.html.erb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<h1>Sessions#create</h1>
```
Comment (lewiscj97) [inline app/views/sessions/create.html.erb:1]: Can be deleted

File: app/views/sessions/login.html.erb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<h1>Sessions#login</h1>
```
Comment (lewiscj97) [inline app/views/sessions/login.html.erb:1]: Can be deleted

File: app/views/sessions/welcome.html.erb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<h1>Welcome</h1>
+<% if logged_in? %>
+    <h1>You are Logged In, <%= current_user.first_name %></h1>
+    <%= button_to "Sign out", '/logout', method: :get%>
+<%end%>
+<%= button_to "Login", '/login', method: :get%>
```
Comment (lewiscj97) [inline app/views/sessions/welcome.html.erb:6]: Only show this if `logged_in?` is false (add to an else)

File: app/views/users/create.html.erb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+<h1>Users#create</h1>
```
Comment (lewiscj97) [inline app/views/users/create.html.erb:1]: Can be deleted

**Review by amin-newcastle — APPROVED** [review]
Great work

**Pull request merged**